.github/workflows/update-custom-image.yml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: "Rebuild All Tagged Custom Cloud Shell Images"
+on:
+  schedule:
+  # run daily at midnight
+    - cron: '0 0 * * *'
```

Comment:
Will this be enough? What happens if Cloud Shell team rebuilds a base image?

Reply:
The Cloud Shell team said they rebuild weekly, and said daily polls should be enough. Right now we don't have anything triggering when they push a new image, but that should be possible to add on the Cloud Build side too now that we have the `cloudbuild.yaml`